Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM alpine:3.8
+FROM golang:1.10-alpine3.8 as build
+RUN apk add --no-cache make
+WORKDIR /go/src/github.com/uswitch/kiam
+ADD . .
+RUN make proto/service.pb.go build-linux
```

Comment:
Could we simplify the Makefile so that the default is to build the binary and have it manage the rest please?